rm -rf build && git reset --hard
patch -Np1 -i "${mypatd:-"../"}/otter-browser.patch"
mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_QTWEBENGINE=OFF -DCMAKE_INSTALL_PREFIX=/usr ..
diff a/CMakeLists.txt b/CMakeLists.txt
index 363c4be..e96b49d 100644
--- a/CMakeLists.txt	1969-12-31 17:00:00.000000000 -0700
+++ b/CMakeLists.txt	1969-12-31 17:00:00.000000000 -0700
@@ -73,7 +73,7 @@ set(CMAKE_CXX_STANDARD 11)
 option(ENABLE_SPELLCHECK "Enable Hunspell based spell checking" ON)
 
 find_package(Qt5 5.6.0 REQUIRED COMPONENTS Core Gui Multimedia Network PrintSupport Qml Svg Widgets)
-find_package(Qt5WebEngineWidgets 5.12.0 QUIET)
+find_package(Qt5WebEngineWidgets 5.9.0 QUIET)
 find_package(Qt5WebKitWidgets 5.212.0 QUIET)
 find_package(Hunspell 1.5.0 QUIET)
 
